runtime/util/xp32/volatile.s

```diff
@@ -129,13 +129,13 @@ longVolatileRead:
         stw r29,168(r1)
         stw r30,172(r1)
         stw r31,176(r1)
-        addi r0,r1,(200+eqSS_longVolatileRead)   #  previous frame = 50 = 45 fixed + 3 extra + 2 added by caller optimized to ternary
+        addi r0,r1,(200+eqSS_longVolatileRead)   #  previous frame = 50 = 45 fixed + 3 extra + 2 added by caller optimized to select
```

Comment:
I don't believe these comments are referring to the "select" IL opcodes.  Please change it back.  Applies to all changes in this file.

Reply:
Thank you for pointing this out, I have pushed the updated commit.